README.md

```diff
@@ -29,6 +29,10 @@ Short description.
 
 ### Vegetation, Agriculture & Food
 
+- [Apeel Sciences](https://apeelsciences.com/) | California, USA | 2012
+
+Apeel Sciences develops plant-derived shelf life extension technology for fresh produce that improves quality and reduces food waste.
```

Comment:
I would rephrase the description to " _develops a technology that consists of a plant-derived shelf for fresh produce that extends its life improves quality and reduces food waste._ ", for better clarity.